python/lsst/ts/adamSensors/model.py

```diff
@@ -26,13 +26,13 @@ def __init__(self, log, simulation_mode=False):
         self.log = log
 
         self.range_size = 20
-        self.range_start = -10 # zero point offset for the ADAM device
+        self.range_start = -10  # zero point offset for the ADAM device
         self.simulation_mode = simulation_mode
 
     def connect(self, ip, port):
         self.clientip = ip
         self.clientport = port
-        if simulation_mode:
+        if self.simulation_mode:
             self.client = None
```

Comment:
Another simulator in the line...

Reply:
Simulating ModbusClient can be another task but I'm not sure if we need it? All the modbus client does is connect and then read values off the Adam's analog inputs. So i have a condition in the read_voltages() method of the model that returns dummy values if simulation mode is on.

Reply:
You probably need it for unit testing and simulation modes.